projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdfActivity.java

```diff
@@ -0,0 +1,234 @@
+package hasadna.noloan2;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Environment;
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.itextpdf.licensekey.LicenseKey;
+import com.itextpdf.html2pdf.*;
```

Comment:
Please don't use * on imports. If there are two * in the imports, given a class, we can't know for sure where it came from just by looking at the code. It also makes some analysis work harder to understand the dependencies :)

Reply:
Got you. Fixed it :)